dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -191,13 +191,17 @@ CRecycleBinEnum::~CRecycleBinEnum()
 
 HRESULT WINAPI CRecycleBinEnum::Initialize(DWORD dwFlags)
 {
-    static LPCWSTR szDrive = L"C:\\";
+    WCHAR szDrive[8];
+    if (GetEnvironmentVariableW(L"SystemDrive", szDrive, _countof(szDrive)))
```

Comment:
This probably should take into account what drives the parent recycle bin is activated for?
But we probably don't support that at all...

Reply:
Registry setting?

Reply:
No, probably needs to be passed from wherever this was invoked from,
but I am not sure we handle recycle bin on a non-system drive at all...